### ISSUE TITLE
International Date Line: map is not fully drawn until first move

### ISSUE THREAD

**Issue by andreynovikov** [mapsforge/vtm#89]
Scroll map to center of Alaska on zoom 3, let application remember the position, close the map and reopen it. Everything to the left of 180 parallel is not drawn until you drag the map. Problem appears in any app - android-example, Cruiser and my own.

**Comment by devemux86** [comment]
Confirmed.

**Comment by Gustl22** [comment]
I wanted to fix this bug, but I don't get further. Btw. seems to appear at any zoom level and independent of tile source.
I set a mass of breakpoints and got to this result:
`TileManager.update()` →→→ _tiles are all loaded without errors_ →→→ `Tilemanager.jobCompleted()` → (1),(2) 

→ (1) `JobComletedEvent.run()` → `Map.events{EventDispatcher}.tell()` → `TileLayer.onMapEvent()` → _leads to no update_

→ (2) `mMap.render()` → _some complicate process_

Before (1) and (2), I set a breakpoint, and sometimes the missing tile is shown at step (1) and sometimes at step (2).

When moving the map afterwards, the other tiles are displayed, without loading new tiles in `TileManager.update()`. So this something has to do with the renderer.
Maybe you have an idea, where this come from?
